scripts/npm.publish.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
The name of the file could be confusing, to actually publishing the scripts or publishing to local registry. May be rename it to `publish_ verdaccio.sh`